geomesa-features/geomesa-feature-avro/src/main/scala/org/locationtech/geomesa/features/avro/FeatureSpecificReader.scala

```diff
@@ -24,28 +24,39 @@ import org.opengis.feature.simple.{SimpleFeature, SimpleFeatureType}
 
 import scala.collection.JavaConversions._
 
-class FeatureSpecificReader(oldType: SimpleFeatureType, newType: SimpleFeatureType,
-                            opts: Set[SerializationOption] = Set.empty)
+class FeatureSpecificReader(var oldType: SimpleFeatureType, var newType: SimpleFeatureType,
```

Comment:
meh not really probably because it was meant originally to be an internal class type thing not exposed to users.